src/main/java/org/sagebionetworks/bridge/sdk/BridgeResearcherClient.java

```diff
@@ -258,15 +258,15 @@ public void deleteUploadSchemaByIdAndRevision(String schemaId, int revision) {
     }
 
     @Override
-    public UploadSchema getUploadSchemaById(String schemaId) {
+    public UploadSchema getUploadSchema(String schemaId) {
         session.checkSignedIn();
         checkArgument(isNotBlank(schemaId), Bridge.CANNOT_BE_BLANK, "schemaId");
         return get(config.getUploadSchemaByIdApi(schemaId), UploadSchema.class);
     }
 
     @Override
-    public ResourceList<UploadSchema> getUploadSchemaForStudy() {
+    public ResourceList<UploadSchema> getUploadSchemasAllRevisions() {
```

Comment:
I feel like getAllUploadSchemas() would be non-ambiguous enough. If not, then getAllUploadSchemasAllRevisions() would probably also work.